### PR TITLE
Cli command for ListTaskListPartition API

### PR DESCRIPTION
**Pull request by anish531213**

### CODE REVIEW

File: tools/cli/taskList.go [hunk 1]
```diff
@@ -43,5 +43,19 @@ func newTaskListCommands() []cli.Command {
...
 			},
 		},
+		{
+			Name:    "list-partition",
+			Aliases: []string{"lp"},
+			Usage:   "List all the tasklist partitions and the hostname for partitions.",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  FlagTaskListTypeWithAlias,
+					Usage: "TaskList description",
```
Comment (yux0) [inline tools/cli/taskList.go]: Update the command usage

File: tools/cli/taskListCommands.go [hunk 2]
```diff
@@ -61,3 +64,44 @@ func DescribeTaskList(c *cli.Context) {
...
+		ErrorAndExit("Operation ListTaskListPartitions failed.", err)
+	}
+	if len(response.DecisionTaskListPartitions) == 0 && len(response.ActivityTaskListPartitions) == 0 {
+		ErrorAndExit(colorMagenta("No partitions for tasklist: "+taskList), nil)
+	}
+	if len(response.DecisionTaskListPartitions) > 0 {
+		printTaskListPartitions("Decision", response.DecisionTaskListPartitions)
+	}
+	if len(response.ActivityTaskListPartitions) > 0 {
+		printTaskListPartitions("Decision", response.ActivityTaskListPartitions)
```
Comment (yux0) [inline tools/cli/taskListCommands.go]: Activity

Comment (anish531213) [reply tools/cli/taskListCommands.go]: fixed.

File: tools/cli/taskListCommands.go [hunk 3]
```diff
@@ -61,3 +64,44 @@ func DescribeTaskList(c *cli.Context) {
...
+		Domain:   common.StringPtr(domain),
+		TaskList: &shared.TaskList{Name: common.StringPtr(taskList)},
+	}
+
+	response, err := frontendClient.ListTaskListPartitions(ctx, request)
+	if err != nil {
+		ErrorAndExit("Operation ListTaskListPartitions failed.", err)
+	}
+	if len(response.DecisionTaskListPartitions) == 0 && len(response.ActivityTaskListPartitions) == 0 {
+		ErrorAndExit(colorMagenta("No partitions for tasklist: "+taskList), nil)
```
Comment (yux0) [inline tools/cli/taskListCommands.go]: Consider not to print error here.

**Pull request merged**